King-Benx/laboremus-challenge
# Ft welcome screen

**Overview:**
This delivers the first screen a user sees.

**Description**
This screen is what users will see before they get to interact with the app fully

**How to Test**
1. Clone the project `git clone git@github.com:King-Benx/laboremus-challenge.git`
2. Checkout to ft-welcome-screen `git checkout ft-welcome-screen`
3. Install dependencies `yarn`
4. Run development server `react-native start`
5. Run the app `react-native run-android`
6. Notice the welcome screen
